docs/guides/external-node/07_snapshots_recovery.md

```diff
@@ -0,0 +1,26 @@
+# Snapshots Recovery
+
+Instead of starting node using DB snapshots, it's possible to configure them to start from a protocol-level snapshots.
+This process is much faster and requires way less storage. Postgres database of a mainnet node recovered from a snapshot
+is only about 300GB.
+
+> [!NOTE]
+>
+> Nodes recovered from snapshot don't have any historical data from before the recovery!
+
+## Configuration
+
+To enable snapshots-recovery on mainnet, you need to set:
```

Comment:
"you need to set the following environment variables"; it might not be clear to a reader that this is the case.

Reply:
corrected